package.nls.json

```diff
@@ -0,0 +1,13 @@
+{
+    "walkthrough.java.runtime": "Make sure you have at least on Java runtime installed.\n[Install New JDK](command:toSide:java.installJdk.fromWalkthrough)",
+
+    "walkthrough.java.projects": "In Java Projects explorer, you can browse your projects, view classpaths and dependencies. \n[View Java Projects](command:javaProjectExplorer.focus)",
+
+    "walkthrough.java.projects.inactive": "In Java Projects explorer, you can browse your projects, view classpaths and dependencies. To get full feautures, you should create or open a folder containing Java projects.\n[Open Command Palette](command:workbench.action.showCommands)\n__Try searching for 'Create Java Project' or 'Open Folder'__",
+
+    "walkthrough.java.debugAndTest":"The debugger will automatically find the entry point of your project, launch it and start debugging. [Learn More...](https://code.visualstudio.com/docs/java/java-debugging)\nThe [Testing View](command:testExplorer.focus) show all the test cases in your project. You can also run/debug your test cases from here. [Learn More...](https://code.visualstudio.com/docs/java/java-testing)\n[Show Run and Debug View](command:workbench.view.debug)",
+
+    "walkthrough.java.extensions": "Additional support for major toolchains, frameworks and application servers.\n[Browse Recommended Extensions](command:toSide:java.extGuide)",
+
+    "walkthrough.java.more": "- Particularly if you want to run your project with a specific runtime, you can [Configure Java Runtime](command:toSide:java.runtime).\n\n- 🎉 Take a look at our [devblogs](akamsLinkToBlogs) for detailed release notes.\n\n- ✨ Discover more shortcuts and features in [Help Center](command:toSide:java.welcome.fromWalkthrough) or check Documentation for tips and troubleshooting."
```

Comment:
If you need to run your project with a specific Java runtime, you can [Configure Java Runtime]
Visit [Dev Blog] to keep up-to-date with our latest product updates and announcements
Discover more shortcuts and features at [Help Center]

Should we also add a link for Github repo to report issues?

Reply:
> Should we also add a link for Github repo to report issues?

Good idea.

Reply:
- If you need to run your project with a specific Java runtime, you can [Configure Java Runtime]
- Visit [Dev Blog] to keep up-to-date with our latest product updates and announcements
- Discover more shortcuts and features at [Help Center]
- Report issues and feedback at [Github repo issues[